modules/core/src/lib/layer.js

```diff
@@ -324,6 +325,13 @@ export default class Layer extends Component {
         attributeManager.invalidateAll();
       }
     }
+
+    if (!this.isComposite && changeFlags.transitionsChanged) {
+      for (const key in changeFlags.transitionsChanged) {
+        // prop changed and transition is enabled
+        this.internalState.transitions.add(key, oldProps[key], props[key], props.transitions[key]);
```

Comment:
Are these `transitions` used for both attributes and uniforms? Or should we call this `propTransitions`?

Reply:
The prop will contain both attribute and uniform transitions.